### DISCUSSION TITLE
Is there any active progress for a mobile/responsive layout for the Tracking UI?

### DISCUSSION THREAD

**Discussion by nnethery** [mlflow/mlflow#6406] [Q&A]
I saw there was a branch made almost 2 years ago that seemed to accomplish this, but it was abandoned. If there isn't any active efforts toward this right now, are there any hurdles why?

**Comment by JHibbard** [comment]
Hi nnethery,

I don't know of any current community efforts towards a mobile/responsive layout for MLflow's UI. Would you be able to share your use cases and maybe file a feature request?

(edit: I can help look into what hurdles there may have been if this is still a useful feature for you.)

Thanks!
Jim
